src/store/selectors.js

```diff
@@ -2,6 +2,8 @@ export const getDataSetList = state => state.datasetReducer.datasets;
 export const getFacetList = state => state.datasetReducer.facets;
 export const getTotalNumberOfDatasets = state => state.datasetReducer.total;
 export const getDataSetError = state => state.datasetReducer.datasetError;
+export const getGlobalErrorState = state => state.presentation.isError;
```

Comment:
Can we name this something like `isGlobalError`?  Just so it's clear that it's a boolean?